app/src/main/res/values/text_styles.xml

```diff
@@ -82,13 +82,13 @@
         <item name="android:textColor">@color/black</item>
     </style>
 
-    <style name="AppTheme.ListTime.Small">
+    <style name="AppTheme.ListTime.Small" parent="AppTheme.ListTime">
```

Comment:
it's not necessary to have a parent-argument when extending from it in the style name. same in the next style

Reply:
Needed to inherit it. 🤷‍♂️